configure/cmake/FindR.cmake

```diff
@@ -11,12 +11,23 @@ set(CMAKE_FIND_APPBUNDLE "NEVER")
 find_program(R_EXEC NAMES R R.exe)
 set(CMAKE_FIND_APPBUNDLE ${TEMP_CMAKE_FIND_APPBUNDLE})
 
+set(ENV{R_HOME} "blah")
```

Comment:
Is this necessary in general or just for being able to reproduce the issue on our local machine? In the latter case, I would remove it.